### DISCUSSION TITLE
Is WSL (Debian) supported or tested?

### DISCUSSION THREAD

**Discussion by the0remora** [retronas/discussions#41] [Q&A]
I'm about to build a streaming rig and while I would rather pass through a GPU to a windows VM I don't have any hardware that supports it. So before I go full VM for RetroNAS is it possible to run RetroNAS from the Debian version of the Windows Subsystem for Linux? I haven't played with WSL enough to even know if I can pass it an ext4 drive

**Comment by sairuk** [comment]
i couldn't say, neither of us (dan nor myself) run Windows so you'd have to try it and see. I don't even have a Windows 10 VM setup anymore.

Looking at the 1.7k open issues at https://github.com/microsoft/WSL/issues you'd probably get lost in the noise even asking there

Its not an officially supported platform so if you run into issues it'd be a best effort approach to fix anything
